tasks/main.yml

```diff
@@ -47,6 +48,26 @@
     - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/packageExtractSyncJob-template.yaml
     - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/solverSyncJob-template.yaml
     - https://raw.githubusercontent.com/thoth-station/kebechet/master/openshift/job-kebechet-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/job-provenanceChecker-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/job-adviser-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/adviser/master/openshift/job-dependencyMonkey-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/amun-api/master/openshift/inspectJob-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/amun-api/master/openshift/inspectJobWithCPU-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/package-extract/master/openshift/job-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/package-analyzer/master/openshift/job-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/adviserSyncJob-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/provenanceCheckerSyncJob-template.yaml
+    - https://raw.githubusercontent.com/thoth-station/graph-sync-job/master/openshift/solverSyncJob-template.yaml
```

Comment:
@harshad16 It looks like we have it on line #49?

Reply:
@bissenbay  hey, thanks for pointing that out :smile: . Fixed it

Reply:
thanks